### PR TITLE
[DC-2369] adding serology qc notebook

### PR DESCRIPTION
**Pull request by kouamea**

### CODE REVIEW

**Comment by kouamea** [comment]
@ksdkalluri

File: data_steward/notebooks/qc_serology_ct_dataset.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+# # Set Up
+
+# parameters
+rt_dataset = ""
+ct_ser_dataset = ""
+new_ct_dataset = ""
+cur_project = ""
+cur_out_project = ""
+
+rt_dataset = 'R2020q4r1_antibody_quest'
```
Comment (ksdkalluri) [inline data_steward/notebooks/qc_serology_ct_dataset.py:25]: These identifiers should be removed.

**Comment by kouamea** [comment]
@ksdkalluri

**Review by ksdkalluri — APPROVED** [review]
Thanks for adding the notebook @kouamea. I will merge this once the integration test passes.

**Pull request merged**